lib/elixir/lib/registry.ex

```diff
@@ -1339,20 +1346,20 @@ defmodule Registry.Supervisor do
   @moduledoc false
   use Supervisor
 
-  def start_link(kind, registry, partitions, listeners, entries) do
-    arg = {kind, registry, partitions, listeners, entries}
+  def start_link(kind, registry, partitions, listeners, entries, compressed) do
+    arg = {kind, registry, partitions, listeners, entries, compressed}
```

Comment:
Seems like we could do some further refactoring to generate this tuple on line 370 and pass it through instead of having the proliferation of args.